service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/UploadC2DocumentsController.java

```diff
@@ -119,21 +113,19 @@ public void handleSubmittedEvent(
         CaseDetails caseDetails = callbackRequest.getCaseDetails();
         CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        if (featureToggleService.isPaymentsEnabled()) {
-
-            if (displayAmountToPay(caseDetails)) {
-                try {
-                    paymentService.makePaymentForC2(caseDetails.getId(), caseData);
-                } catch (FeeRegisterException | PaymentsApiException ignore) {
-                    applicationEventPublisher.publishEvent(new FailedPBAPaymentEvent(callbackRequest, C2_APPLICATION));
-                }
-            }
 
-            if (NO.getValue().equals(caseDetails.getData().get(DISPLAY_AMOUNT_TO_PAY))) {
+        if (displayAmountToPay(caseDetails)) {
+            try {
+                paymentService.makePaymentForC2(caseDetails.getId(), caseData);
+            } catch (FeeRegisterException | PaymentsApiException ignore) {
                 applicationEventPublisher.publishEvent(new FailedPBAPaymentEvent(callbackRequest, C2_APPLICATION));
             }
         }
 
+        if (NO.getValue().equals(caseDetails.getData().get(DISPLAY_AMOUNT_TO_PAY))) {
```

Comment:
this could just be an else on the above if

Reply:
done